articles/tokens/concepts/refresh-token-rotation.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
Doesn't it make sense add a new PKCE section on the page https://auth0.com/docs/architecture-scenarios/spa-api/part-1#implicit-grant, detailing the flow with a rotating Refresh Token?

Reply:
It does make sense. Can you provide that detail @dctoon ?

Reply:
Got it via Slack - thanks Toon!